README.md

```diff
@@ -2,8 +2,7 @@ VTEX Minicart
 =====
 
 ## Description
-The VTEX minicart app shows a list of all items that a customer add in our Checkout OrderForm API. 
-This is a VTEX app that is used by Dreamstore product.
+The VTEX minicart app shows a list of all items that a customer add in our Checkout OrderForm API, and this app is used by Dreamstore product.
```

Comment:
` Checkout OrderForm API and this` :hammer: 
No need for the `,`

Reply:
this is a nit, I prefer the suggestion of Grammarly.